### ISSUE TITLE
Error:  'str' object has no attribute 'get'

### ISSUE THREAD

**Issue by sviluppatoreLaRegione** [karthiks3000/postman-doc-gen#18]
I get this error:

```
$ ./postman_doc_gen ./api.test.postman_collection.json -o ./api-docs
Traceback (most recent call last):
  File "postman_doc_gen.py", line 29, in <module>
  File "document_generator.py", line 63, in generate_doc
  File "document_generator.py", line 204, in add_items
  File "document_generator.py", line 220, in add_apis
  File "document_generator.py", line 347, in get_body
  File "document_generator.py", line 359, in get_key_values
AttributeError: 'str' object has no attribute 'get'
[31954] Failed to execute script postman_doc_gen
```

**Comment by karthiks3000** [comment]
@sviluppatoreLaRegione  can you provide a sample collection to reproduce the error?

**Comment by sviluppatoreLaRegione** [comment]
Check here https://gist.github.com/sviluppatoreLaRegione/5a0d14cf2ebec961631bbed89d06f03c
This is the same collection that I used.

**Comment by rudyzeinoun** [comment]
You are probably using an old Postman version that is exporting the JSON file in Postman v2.0 format. Upgrade to the latest version of Postman and export using v2.1 format.

**Comment by sviluppatoreLaRegione** [comment]
@rudyzeinoun if you check my JSON you can see that version is 2.1

**Comment by ctrlsam** [comment]
I'm getting this too

**Comment by tudor33sud** [comment]
Same here on v2.1 collection

**Comment by l-tree** [comment]
I was getting a similar error using the version 2.0 export, which was resolved when I used version 2.1.

The original error:
`[26200] Failed to execute script postman_doc_gen 
Traceback (most recent call last):
  File "postman_doc_gen.py", line 29, in <module>
  File "document_generator.py", line 63, in generate_doc
  File "document_generator.py", line 204, in add_items
  File "document_generator.py", line 233, in add_apis
AttributeError: 'str' object has no attribute 'get'
`

**Comment by greenbrettmichael** [comment]
I am also having this issue, as a workaround I added the following change to document generation
 def get_key_values(item_list: json) -> list:
        if item_list is None or len(item_list) == 0:
            return None

        key_value_list = []

        for item in item_list:
            try:
                value_string = DocumentGenerator.escape_string(item.get(VALUE))
                if type(item.get(DESCRIPTION)) == str:
                    desc_string = DocumentGenerator.escape_string(item.get(DESCRIPTION))
                else:
                    desc_string = DocumentGenerator.escape_string(
                        item.get(DESCRIPTION, {}).get(CONTENT, None))

                key_value_list.append(
                    KeyValueModel(item.get(KEY), value_string, desc_string)
                )
            except Exception as e:
                print(e)
                print("could not add key value")
                print(item)
        return key_value_list
